.github/workflows/ci-backend-cql.yml

```diff
@@ -87,6 +87,18 @@ jobs:
           - module: cql
             args: "-Pcassandra3-murmur-client-auth -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
             name: murmur-client-auth
+          - module: cql
+            args: "-Pscylladb -Dtest=\"**/diskstorage/cql/*\""
+            name: scylladb-diskstorage
+          - module: cql
+            args: "-Pscylladb -Dtest=\"**/graphdb/cql/*\""
+            name: scylladb-graphdb
+          - module: cql
+            args: "-Pscylladb -Dtest=\"**/hadoop/*\""
```

Comment:
This one seems to be problematic. https://github.com/JanusGraph/janusgraph/pull/2352/checks?check_run_id=2032069211

Previously the [build](https://github.com/JanusGraph/janusgraph/runs/2019866227) was passing because the maven surefire plugin does not include *IT test classes by default.